crates/driver/src/tests/cases/protocol_fees.rs

```diff
@@ -50,23 +50,35 @@ async fn protocol_fee_test_case(test_case: TestCase) {
         .sell_amount(test_case.execution.solver.sell)
         .buy_amount(test_case.execution.solver.buy);
     let pool = ab_adjusted_pool(quote);
-
+    let solver_fee = test_case.execution.driver.sell / 100;
+    let executed = match test_case.order.side {
+        order::Side::Buy => (test_case.order.buy_amount > test_case.execution.solver.buy)
+            .then_some(test_case.execution.solver.buy),
+        order::Side::Sell => {
+            if test_case.order.sell_amount > test_case.execution.solver.sell {
```

Comment:
nit: why use `.then_some()` in one case and `if else` in another?

Reply:
Forgot to update it, thanks